app/views/fileChecksFailed.scala.html

```diff
@@ -20,16 +20,28 @@ <h2 class="govuk-error-summary__title" id="error-summary-title">
                 </a>
               </p>
             } else {
-              <p class="govuk-body">One or more files you uploaded have failed our checks. Please
-                <a class="govuk-link" href="@routes.ContactController.contact()">contact</a>
-                us to resolve this issue.
-              </p>
-              <p class="govuk-body">Possible failure causes:</p>
-              <ul class="govuk-list govuk-list--bullet">
-                <li>Password protected files</li>
-                <li>Zip files</li>
-                <li>Corrupted files</li>
-              </ul>
+              @if(fileStatus == "PasswordProtected"){
+                <p class="govuk-body govuk-!-font-weight-bold">Your folder contains one or more password protected files.</p>
+                <p>We cannot accept password protected files. Once removed or replaced, try uploading your folder again.</p>
+              } else { @if(fileStatus == "Zip") {
+                <p class="govuk-body govuk-!-font-weight-bold">Your folder contains one or more zip files.</p>
+                <p>We cannot accept zip files and similar archival package file formats.
+                  These commonly have file extensions such as .zip, .iso, .7z, .rar and others.
+                  please see our <a class="govuk-link" href="@routes.FaqController.faq()" target="_blank" rel="noreferrer noopener">FAQ(Opens in new tab)</a> for a full list.
+                  Either remove or unpack your zip and archival package files and try uploading again.</p>
+                } else {
+                  <p class="govuk-body">One or more files you uploaded have failed our checks. Please
+                    <a class="govuk-link" href="@routes.ContactController.contact()">contact</a>
+                    us to resolve this issue.
+                  </p>
+                  <p class="govuk-body">Possible failure causes:</p>
+                  <ul class="govuk-list govuk-list--bullet">
```

Comment:
Not sure what the designs have now, but wondering whether this list is needed anymore, given we're capturing password protected and zip error separately.